packages/codec/docs/tsconfig.json

```diff
@@ -9,7 +9,8 @@
     "outDir": "dist",
     "baseUrl": "..",
     "lib": [
-      "es2017"
+      "es2017",
+      "dom"
```

Comment:
As mentioned [here](https://github.com/web3/web3.js/issues/6011#issuecomment-1532617232), `dom` was added to handle the use of `RequestInit`, but there should be a better way to handle this on our end